www-servers/nginx-unit/nginx-unit-1.3-r1.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="A dynamic web and application server"
+HOMEPAGE="https://unit.nginx.org"
+SRC_URI="https://unit.nginx.org/download/unit-${PV}.tar.gz -> ${P}.tar.gz"
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64"
+
+IUSE="perl python ruby"
+REQUIRED_USE="|| ( ${IUSE} )"
+DEPEND="perl? ( dev-lang/perl:= )
+	python? ( dev-lang/python:= )
+	ruby? ( dev-lang/ruby:= )"
+RDEPEND="${DEPEND}"
+S="${WORKDIR}/unit-${PV}"
+
+src_configure() {
+	./configure \
+		--prefix=/usr \
+		--log=/var/log/${PN} \
+		--state=/var/lib/${PN} \
+		--pid=/run/${PN}.pid \
+		--control=unix:/run/${PN}.sock || die "Core configuration failed"
+	for flag in ${IUSE} ; do
+		if use ${flag} ; then
+			./configure ${flag} || die "Module configuration failed: ${flag}"
+		fi
+	done
+}
+
+src_install() {
+	default
+	keepdir /var/lib/${PN}
+	fperms 0770 /var/lib/${PN}
```

Comment:
Use `diropts` to control mode when creating it, instead of changing post-creation.